groovy/postInit/components/Batteries.groovy

```diff
@@ -223,54 +156,422 @@ mods.gregtech.assembler.recipeBuilder()
+mods.gregtech.extractor.removeByInput(2, [metaitem('battery.re.mv.sodium')], null)
+
+ASSEMBLER.recipeBuilder()
+        .circuitMeta(5)
+        .inputs(ore('plateStainlessSteel') * 4)
+        .inputs(metaitem('cableGtSingleCopper') * 2)
+        .fluidInputs(fluid('plastic') * 432)
+        .outputs(metaitem('battery.hull.mv'))
+        .duration(160)
+        .EUt(120)
+        .buildAndRegister()
+ASSEMBLER.recipeBuilder()
+        .circuitMeta(5)
+        .inputs(ore('plateStainlessSteel') * 4)
+        .inputs(metaitem('cableGtSingleAnnealedCopper') * 2)
+        .fluidInputs(fluid('plastic') * 432)
+        .outputs(metaitem('battery.hull.mv'))
+        .duration(160)
+        .EUt(120)
+        .buildAndRegister()
+RecyclingHelper.handleRecycling(metaitem('battery.hull.mv'),
+        [ore('plateStainlessSteel') * 4, metaitem('cableGtSingleCopper') * 2, ore('dustPlastic') * 3])
+
+// HV Battery Hull
+mods.gregtech.assembler.removeByInput(4, [metaitem('cableGtSingleGold') * 4, metaitem('plateBatteryAlloy') * 9], [fluid('plastic') * 1296 * 1296])
+mods.gregtech.extractor.removeByInput(2, [metaitem('battery.re.hv.cadmium')], null)
+mods.gregtech.extractor.removeByInput(2, [metaitem('battery.re.hv.lithium')], null)
+mods.gregtech.extractor.removeByInput(2, [metaitem('battery.re.hv.sodium')], null)
+
+ASSEMBLER.recipeBuilder()
+        .circuitMeta(5)
+        .inputs(ore('plateStainlessSteel') * 8)
+        .inputs(metaitem('cableGtSingleGold') * 4)
+        .fluidInputs(fluid('plastic') * 1296)
+        .outputs(metaitem('battery.hull.hv'))
+        .duration(240)
+        .EUt(120)
+        .buildAndRegister()
+RecyclingHelper.handleRecycling(metaitem('battery.hull.hv'),
+        [ore('plateStainlessSteel') * 8, metaitem('cableGtSingleGold') * 4, ore('dustPlastic') * 9])
+
+// EV Battery Hull
+mods.gregtech.assembler.removeByInput(480, [metaitem('cableGtSingleAluminium') * 2, metaitem('plateBlueSteel') * 2], [fluid('polytetrafluoroethylene') * 144 * 144])
+mods.gregtech.extractor.removeByInput(2, [metaitem('battery.ev.vanadium')], null)
+
+ASSEMBLER.recipeBuilder()
+        .circuitMeta(5)
+        .inputs(ore('plateStainlessSteel') * 16)
+        .inputs(metaitem('cableGtSingleAluminium') * 2)
+        .fluidInputs(fluid('polytetrafluoroethylene') * 144)
+        .outputs(metaitem('battery.hull.ev'))
+        .duration(240)
+        .EUt(480)
+        .buildAndRegister()
+RecyclingHelper.handleRecycling(metaitem('battery.hull.ev'),
+        [ore('plateStainlessSteel') * 16, metaitem('cableGtSingleAluminium') * 2, ore('dustPolytetrafluoroethylene')])
+
+// IV Battery Hull
+mods.gregtech.assembler.removeByInput(1920, [metaitem('cableGtSinglePlatinum') * 2, metaitem('plateRoseGold') * 6], [fluid('polytetrafluoroethylene') * 288 * 288])
+mods.gregtech.extractor.removeByInput(2, [metaitem('battery.iv.vanadium')], null)
+
+ASSEMBLER.recipeBuilder()
+        .circuitMeta(5)
+        .inputs(ore('plateStainlessSteel') * 32)
```

Comment:
why does an IV battery use stainless steel?

Reply:
There are 2 hull types: primitive (for early LV & MV batteries, made from battery alloy) and the main one.

I'm sure about using stainless steel for HV & LV hull (as the only battery using the LV hull is Li-Ion gated HV).
As for MV, this hull type is used by NiZn (gated HV), NiMH (uncraftable - needs La) and NiCd (late MV gate because of Cd). So, using stainless steel IMO fits it's purpose.

As for EV/IV, I'm not that sure. Maybe we can use frame instead of plates (to show that that's a big battery assembly), maybe we can use titanium-based material or one of the recently added superalloys. Should be discussed.

Reply:
No. Should it?